python/py-pytest-html/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       pytest-html
+name                py-${real_name}
```

Comment:
please just set the name to ```py-pytest-html``` and remove the line above; there is no need for another variable and it will simplify stuff later on (the python PG sets ```python.rootname``` to everything after ```py-```).

Reply:
Thanks done.